docs/deepops/dgx-pod.md

```diff
@@ -203,8 +203,6 @@ note: The following K8s services are especially helpful for air-gapped environem
 
 Provision DGX nodes with the official DGX ISO over PXE boot using DGXie.
 
-> Note: The `scripts/do_ipmi.sh` script has these commands and can be looped over multiple hosts
```

Comment:
Did we get rid of this script altogether? I thought we referenced it somewhere in the dgxie workflow.

Reply:
I just removed the script since the commands in it are referenced in some docs